openfecwebapp/templates/datatable.html

```diff
@@ -2,7 +2,7 @@
 {% import 'macros/page-header.html' as header %}
 
 {% set breadcrumbs_title=title %}
-{% set breadcrumbs=[('', 'Advanced data')] %}
+{% set breadcrumbs=[(url_for('advanced'), 'Advanced data')] %}
```

Comment:
Do we want to disable this for now since we don't have the content for this page?